client/containers/DashboardSubmissions/arbitrationMenu.scss

```diff
@@ -0,0 +1,6 @@
+.arbitration-menu {
+	display: grid;
```

Comment:
I wonder whether a:

```css
display: flex;
> *:not(:last-child) {
    margin-right: 40px;
}
```
might do the job better here. That way you don't have to specify the `gridColumn` in React, and it scales to support more (or fewer) buttons, which eliminates this kind of goofy state:

![image](https://user-images.githubusercontent.com/2208769/149993827-d0fd68a4-72a0-4c4c-8caf-b107a8f8b8a4.png)

Reply:
I'm happy either way. In the `all` filter view with `grid` the delete / decline / approve buttons stay vertically aligned by effect tolerant of the missing buttons, but at the cost of the goofy state you show above. Comparison, from the figma mock ups:
![Screenshot from 2022-01-24 01-55-12](https://user-images.githubusercontent.com/14115194/150735836-c694c60f-63e9-43f8-bc09-bccb4eb64260.png)

Reply:
That's a great point! In that case, the "delete" button should definitely go at the end, though, since it's always visible. I think it should also use the `trash` icon, because the `x` is mostly used to mean "Remove from Collection" elsewhere and folks may assume that's the action here, too.